generators/search-map.js

```diff
@@ -4,7 +4,8 @@ var fs = require('fs'),
 	Q = require('q'),
 	writeFile = Q.denodeify(fs.writeFile),
 	mkdirs = Q.denodeify(require("fs-extra").mkdirs),
-	sanitizeHtml = require('sanitize-html');
+	markdown = require('github-flavored-markdown'),
```

Comment:
Ok, let’s use the new package and also file an issue in `bit-docs-generate-html` to start using the new package too.